src/interpreter/Node.h

```diff
@@ -44,7 +44,7 @@ class Node;
 
 namespace interpreter {
 class ViewContext;
-class RelationWrapper;
+struct RelationWrapper;
```

Comment:
This change seems unrelated to this PR. It would be much better to put unrelated changes into another PR. That said, I think a better code change would be to leave this as `class` (an improvement in readability over having to think about why `struct` was used) and change the class itself to use `class` rather than `struct`.

Reply:
I fixed this but could we avoid forward references.